client/sites/marketing/tools/marketing-features-data.ts

```diff
@@ -0,0 +1,100 @@
+import page from '@automattic/calypso-router';
+import { localizeUrl } from '@automattic/i18n-utils';
+import { translate } from 'i18n-calypso';
+import fiverrLogo from 'calypso/assets/images/customer-home/fiverr-logo.svg';
+import rocket from 'calypso/assets/images/customer-home/illustration--rocket.svg';
+import earnIllustration from 'calypso/assets/images/customer-home/illustration--task-earn.svg';
+import wordPressLogo from 'calypso/assets/images/icons/wordpress-logo.svg';
+import { marketingConnections } from 'calypso/my-sites/marketing/paths';
+import * as T from 'calypso/types';
+import { MarketingToolsFeatureData } from './types';
+
+export const getMarketingFeaturesData = (
+	selectedSiteSlug: T.SiteSlug | null,
+	recordTracksEvent: ( event: string ) => void
+): MarketingToolsFeatureData[] => [
+	{
+		title: translate( 'Let our WordPress.com experts build your site!' ),
```

Comment:
I would move the `translate` functions to the component. What do you think?

Reply:
Thinking about it, that would allow this list to be _mostly_ raw data. We can even extract the `onClick` handlers, but that could be done when the specific use case arises. Currently, extracting the `translate` and `localizeUrl` is a good approach. I will go for it.

Reply:
Moved to the main component and successfully tested as part of 361fd58